CMakeLists.txt

```diff
@@ -532,16 +537,17 @@ file(GENERATE OUTPUT ${CONFIG_PATH} CONTENT
 ")
 
 if(RUN_TESTS)
+  add_executable(devilutionx-tests WIN32 MACOSX_BUNDLE ${devilutionxtest_SRCS})
   include(CTest)
   include(GoogleTest)
   find_package(GTest REQUIRED)
-  include_directories(${GTEST_INCLUDE_DIRS})
-  add_definitions(-DRUN_TESTS)
+  target_include_directories(devilutionx-tests PRIVATE ${GTEST_INCLUDE_DIRS})
   if(NOT CMAKE_CXX_COMPILER_ID MATCHES "MSVC")
-    add_definitions(-fprofile-arcs -ftest-coverage)
+    target_compile_definitions(devilutionx-tests PRIVATE -fprofile-arcs -ftest-coverage)

```

Comment:
This looks wrong, and it was wrong before. It's defining a macro constant `-fprofile-arcs` etc.

Perhaps `target_compile_options`?

Reply:
👍  Let's give it a try. I changed it.

Reply:
This works now. Thanks 🙂